docs/console/tutorials/configure-marketplace-components/api-portal.mdx

```diff
@@ -45,62 +43,46 @@ Moreover, you need at least:
 
-The first plugin we are going to create is the _API portal_. The steps to create it are:
+and the following endpoints:
+- _/documentations/api-portal_ to the [_API Portal_](/runtime_suite/api-portal/10_overview.md) service 
+- _/documentations/api-portal/api_ to the [_Swagger Aggregator_](/runtime_suite/swagger-aggregator/10_overview.md) service
+- _/documentations/openapi_ to the [_Swagger Aggregator_](/runtime_suite/swagger-aggregator/10_overview.md) service
+- _/documentations/swagger_ to the [_Swagger Aggregator_](/runtime_suite/swagger-aggregator/10_overview.md) service
 
-1. Click on the _Create a Microservice_ button;
-1. In the dropdown menu, select _From Marketplace_ option;
-1. On the right side, you will see a catalogue of plugin, templates and examples;
-1. Type `API portal` in the search bar;
-1. Select the **API portal** card from the list of result;
-1. Click on the _Create_ button applying the default fields.
-1. Save the changes by committing. If you do not know how to do it, take a quick look at [Mia-Platform handbook](/console/handbooks/project-usage.md#save-changes) first.
-
-![Create API portal](img/create_api_portal.png)
-
-### Swagger Aggregator
-
-In order to create this plugin:
-
-1. Click on the _Create a Microservice_ button;
-1. In the dropdown menu, select _From Marketplace_ option;
-1. On the right side, you will see a catalogue of plugin, templates and examples;
-1. Type **Swagger Aggregator** in the search bar;
-1. Select the card from the list of result;
-1. Click on the _Create_ button applying the default fields.
-1. Save the changes by committing.
+:::caution
+In the Mia-Platform Marketplace are available two different applications to install the _API Portal_ and _Swagger Aggregator_.
+The difference between them is the _API Gateway_ used: 
+- the _API Documentation Aggregator_ uses the _Envoy API Gateway_
+- the _API Documentation Aggregator NGINX_ uses the _NGINX API Gateway_
 
-![Create API portal](img/create_swagger_aggregator.png)
+Pay attention and choose the one best suites your needs.
+:::
 
-## Configure plugins endpoints
+### Install the application
 
-Now we need to create the necessary endpoints that are required to make the whole system work. This consists of going to the _endpoints_ section and creating:
+To install our _API Documentation Aggregator_ follow the following steps:
 
-1. `/documentations/api-portal` having the following configuration:
-   - **Base path**: `/documentations/api-portal`;
-   - **Type**: "_Microservice_";
-   - **Microservice**: "api-portal";
-   - **Rewrite Base Path**: `/`;
-1. `/documentations/api-portal/api` having the folowing configuration:
-   - **Base path**: `/documentations/api-portal/api`;
-   - **Type**: "_Microservice_";
-   - **Microservice**: "swagger-aggregator";
-   - **Rewrite Base Path**: `/`;
+1. Click the _Applications_ menu item in the _Workloads_ section of the console left menu;
+2. Click the button _Create new Application_;
```

Comment:
leave all numbers to `1.` the markdown renderer will set-up the correct numbers. If you happen to add a new item in-between you would have to change all the numbers

Reply:
Done